Formula/vulkan-extensionlayer.rb

```diff
@@ -0,0 +1,66 @@
+class VulkanExtensionlayer < Formula
+  desc "Layer providing Vulkan features when native support is unavailable"
+  homepage "https://github.com/KhronosGroup/Vulkan-ExtensionLayer"
+  url "https://github.com/KhronosGroup/Vulkan-ExtensionLayer/archive/refs/tags/v1.3.246.tar.gz"
+  sha256 "6ce8e35dd56d46f308f4581a4d0cdfd660b9aa733a4271442365fda422d4227b"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-ExtensionLayer.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "googletest" => :build
+  depends_on "python@3.11" => :build
+  depends_on "vulkan-loader" => :test
+  depends_on "vulkan-tools" => :test
+  depends_on "glslang"
+  depends_on "spirv-headers"
+  depends_on "spirv-tools"
+  depends_on "vulkan-headers"
+
+  on_linux do
+    depends_on "libxcb" => :build
+    depends_on "libxrandr" => :build
+    depends_on "mesa" => :build
+    depends_on "pkg-config" => :build
+    depends_on "wayland" => :build
+  end
+
+  def install
+    system "cmake", "-S", ".", "-B", "build",
+           "-DGLSLANG_INSTALL_DIR=#{Formula["glslang"].prefix}",
+           "-DSPIRV_HEADERS_INSTALL_DIR=#{Formula["spirv-headers"].prefix}",
+           "-DSPIRV_TOOLS_INSTALL_DIR=#{Formula["spirv-tools"].prefix}",
+           "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].prefix}",
+           *std_cmake_args
+    system "cmake", "--build", "build"
+    system "cmake", "--install", "build"
+  end
+
+  def caveats
+    <<~EOS
+      In order to use these layers in a Vulkan application, you may need to place them in the environment with
+        export VK_LAYER_PATH=#{opt_share}/vulkan/explicit_layer.d
+    EOS
+  end
+
+  test do
+    ENV.prepend_path "VK_LAYER_PATH", share/"vulkan/explicit_layer.d"
+    ENV["VK_ICD_FILENAMES"] = Formula["vulkan-tools"].lib/"mock_icd/VkICD_mock_icd.json"
+
+    ver = Formula["vulkan-headers"].version
+    expected = <<~EOS
+      Instance Layers: count = #{build.head? ? 3 : 2}
+      --------------------------
+    EOS
+    if build.head?
+      expected += <<~EOS
+        VK_LAYER_KHRONOS_shader_object      Shader object layer              #{ver}  version 1
+      EOS
+    end
+    expected += <<~EOS
```

Comment:
```suggestion
      Instance Layers: count = 2
      --------------------------
    EOS
```
Let's make this simpler -- we don't test `head` in CI.

Reply:
Okay!  Is there a way to remind me—or whoever updates the formula next—that the next released version will introduce `VK_LAYER_KHRONOS_shader_object` and so the test will need to be updated?

Reply:
Trying something for this.